tests/rustdoc-ui/doctest/main-alongside-stmts.rs

```diff
@@ -15,6 +15,7 @@
 //@ check-pass
 
 //! ```
+//~^ WARN the `main` function of this doctest won't be run
```

Comment:
You could also rewrite it as

```suggestion
//~v WARN the `main` function of this doctest won't be run
//! ```
```

To preserve the stderr output I believe but ultimately it doesn't matter, all of these spans are super rough anyway unfortunately.